iidm/iidm-converter-api/src/test/java/com/powsybl/iidm/import_/ImportersTest.java

```diff
@@ -145,8 +146,8 @@ public void setPostProcessor() {
     }
 
     @Test
-    public void importData() {
-        Network network = Importers.importData(loader, TEST_FORMAT, null, null, computationManager, importConfigMock);
+    public void importData() throws IOException {
+        Network network = Importers.loadNetwork((ReadOnlyDataSource) null);
```

Comment:
Probably has nothing to do with this PR, but what is the purpose of this test?????
Why are we allowing a null data source and then check that a mock importer, we have succeed to import a network?

Reply:
Maybe it would worth a fix but could be done in another PR, as you wish

Reply:
I think it was to test if the importer was found in the LOADER and not the importer itself (it tests the mechanisms to find an importer).

Reply:
> I think it was to test if the importer was found in the LOADER and not the importer itself (it tests the mechanisms to find an importer).

In any case null datasource should throws an exception even in such a test

Reply:
I'm writing an issue about it